tests/test_core.py

```diff
@@ -8,15 +10,52 @@ class test_Interpolator(unittest.TestCase):
 
     def test_init(self):
         """Test creation from data."""
-        raise NotImplementedError()
+
+        #test construction with tuple input
+        f = relentless.core.Interpolator(x=(-1,0,1), y=(-2,0,2))
+        self.assertEqual(f._domain, (-1,1))
+        self.assertIsInstance(f._spline, Akima1DInterpolator)
```

Comment:
This is a good thought, but we don't want to be this specific about what the exact implementation of `_spline` is, in case it changes in future. We jalso ust want to test the exposed interface. So, you should check `f.domain` rather than `f._domain`, and not check the instance of `_spline`.